packages/plugin-ext/src/main/browser/view/tree-views-main.ts

```diff
@@ -108,6 +108,23 @@ export class TreeViewsMainImpl implements TreeViewsMain, Disposable {
         }
     }
 
+    async $setMessage(treeViewId: string, message: string): Promise<void> {
+        this.viewRegistry.getView(treeViewId).then(viewPanel => {
+            const widget = viewPanel && viewPanel.widgets[0];
+            if (widget instanceof TreeViewWidget) {
+                widget.message = message;
```

Comment:
Yes I've followed the patterns of existing methods. And actually I don't see any valuable error processing here other than just ignoring in case of absent tree view.